src/main/java/com/cottonfactory/products/controllers/sweater/SweaterController.java

```diff
@@ -0,0 +1,19 @@
+package com.cottonfactory.products.controllers.sweater;
+
+import com.cottonfactory.products.entities.sweater.Sweater;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class SweaterController {
```

Comment:
Add the `@Api` annotation. It'll make your work appear in the Swagger Docs. Check the readme for more details!